addons/im_livechat/data/im_livechat_channel_demo.xml

```diff
@@ -23,118 +23,133 @@
             <field name="livechat_operator_id" ref="base.partner_admin"/>
             <field name="name">Visitor #234, Mitchell Admin</field>
             <field name="anonymous_name">Visitor #234, Mitchell Admin</field>
+            <field name="channel_last_seen_partner_ids" eval="[(0, 0, {'partner_id': ref('base.partner_admin')}), (0, 0, {'partner_id': ref('base.public_partner')})]"/>
         </record>
         <record id="im_livechat_rating_1" model="rating.rating">
             <field name="res_id" ref="im_livechat.im_livechat_mail_channel_data_0"/>
             <field name="res_model_id" ref="mail.model_mail_channel"/>
             <field name="rated_partner_id" ref="base.partner_admin"/>
             <field eval="False" name="partner_id"/>
             <field eval="True" name="consumed"/>
+            <field name="partner_id" ref="base.public_partner"/>
         </record>
-        <function eval="([ref('im_livechat.im_livechat_mail_channel_data_0')], 5)" model="mail.channel" name="rating_apply"/>
-        <record id="im_livechat_mail_message_5_data" model="mail.message">
+        <record id="im_livechat_mail_message_1_data" model="mail.message">
```

Comment:
While we are at it, I would suggest that we rename those xml ids to contain both the channel id and the message id in the name, that would make it much easier to see which message belongs to which conversation when using ref.

Reply:
Hello, I think you forgot to rename those of channel_0?